src/endpoints/endpoints.service.ts

```diff
@@ -1,4 +1,41 @@
-import { Injectable } from '@nestjs/common';
+import { BadRequestException, Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { ZaLaResponse } from 'src/common/helpers/response';
+import { Endpoint } from 'src/entities/endpoint.entity';
+import { Repository } from 'typeorm';
+import { FindApiEndpoints } from './dto/find-api-endpoints.dto';
+
 
 @Injectable()
-export class EndpointsService {}
+export class EndpointsService {
+    constructor(
+        @InjectRepository(Endpoint)
+        private readonly endpointRepo : Repository<Endpoint>
+    ){}
+
+    async getAllApiEndpoints(dto: FindApiEndpoints){
+        try {
+            //check if api exists in Endpoint table
+            const apiExists = await this.endpointRepo.find({where : { apiId : dto.apiId }})
```

Comment:
why are you creating a dto for the api_id?  You should get the id as a params, dto should not be required

Reply:
Done. Thanks